include/gz/sim/Server.hh

```diff
@@ -338,6 +338,9 @@ namespace gz
       /// \brief Stop the server. This will stop all running simulations.
       public: void Stop();
 
+      /// \brief Reset all runners in this simulation
+      public: void ResetAll();
```

Comment:
Any reason to not offer a `Reset` function that takes a `_worldIndex` similar to the other APIs in this class?